timerapplet/src/timerapplet.c

```diff
@@ -423,10 +429,16 @@ timer_applet_fill (MatePanelApplet* applet_widget)
     g_signal_connect (applet->applet, "destroy",
                       G_CALLBACK (timer_applet_destroy),
                       applet);
-
```

Comment:
This blank line shouldn't be removed.

Reply:
Brought the missing blank line back in latest force-push, thanks